core/quantity/src/test/BearingAzimuth.test.ts

```diff
@@ -176,6 +176,168 @@ describe("Bearing format tests:", () => {
+      { input: "n85 45 45.9e", expected: 85.76275 },
+      { input: "n85 45 45e", expected: 85.7625 },
+      { input: "n85 60 60e", expected: 86.0167 },
+      { input: "n85 45 65e", expected: 85.76275 },
+      { input: "s65:40:00w", expected: 245.667 },
+      { input: "s45e", expected: 135 },
+      { input: "S45E", expected: 135 },
+      { input: "s45.45e",    expected: 134.25 },
+      { input: "s45 45 00e", expected: 134.25 },
+      { input: "s 45 45 00 e", expected: 134.25 },
+      { input: "s45d45m45se", expected: 134.2375 },
+      { input: "s45d45m45.0se", expected: 134.2375 },
+      { input: "s45d45m45.0e", expected: 134.2375 },
+      { input: "NE", expected: 45.0 },
+      { input: "SE", expected: 135.0 },
+      { input: "SW", expected: 225.0 },
+      { input: "NW", expected: 315.0 },
+      { input: "N45E", expected: 45.0 },
+      { input: "S45W", expected: 225.0 },
+      { input: "N45W", expected: 315.0 },
+      { input: "S45E", expected: 135.0 },
+      { input: "nE", expected: 45.0 },
+      { input: "sW", expected: 225.0 },
+      { input: "nW", expected: 315.0 },
+      { input: "sE", expected: 135.0 },
+      { input: "n45e", expected: 45.0 },
+      { input: "s45w", expected: 225.0 },
+      { input: "n45w", expected: 315.0 },
+      { input: "s45e", expected: 135.0 },
+      { input: "N", expected: 0.0 },
+      { input: "S", expected: 180.0 },
+      { input: "n", expected: 0.0 },
+      { input: "s", expected: 180.0 },
+    ];
+
+    for (const { input, expected } of validTestData) {
+      const result = Parser.parseQuantityString(input, bearingParser);
+      if (!Parser.isParsedQuantity(result)) {
+        expect.fail(`Expected a parsed quantity for input "${input}"`);
+      }
+      expect(result.value).to.be.closeTo(expected, 0.01);
+    }
+    const unsupportedInputs = [
+      "s45+45+45e",
+      "s45/45/45e",
+      "s45*45*45e",
+      "s45-45-45e",
+    ];
+    for (const input of unsupportedInputs) {
+      const result = Parser.parseQuantityString(input, bearingParser);
+      expect(Parser.isParsedQuantity(result)).to.be.false;
+    }
+  });
+
+  it("should correctly parse breaing using RAD units", async () => {
+    const bearingRadJson: FormatProps = {
+      minWidth: 2,
+      precision: 0,
+      type: "Bearing",
+      revolutionUnit: "Units.REVOLUTION",
+      composite: {
+        includeZero: true,
+        units: [
+          { name: "Units.RAD" },
+        ],
+      },
+    };
+
+    const bearingRad = new Format("BearingRad");
+    await bearingRad.fromJSON(unitsProvider, bearingRadJson);
+    expect(bearingRad.hasUnits).to.be.true;
+
+    const bearingRadParser = await ParserSpec.create(bearingRad, unitsProvider, rad);
+
+    const testData = [
+      { input: "N0.785398E", expected: 0.785398 }, // 45 degrees in radians
+      { input: "S0.785398E", expected: 2.35619 }, // 135 degrees in radians
+      { input: "S", expected: 180 }, // 180 degrees in radians
```

Comment:
this doesn't make sense, the bearingRadParser is using radian as the output. Shouldn't the expected value be 3.14159?